.travis/setupOpenShift.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eoux pipefail
+
+export OPENSHIFT_VERSION=v3.9
```

Comment:
We should start with 4.1, and I think there's no `oc cluster up` for that.

Reply:
I don't think 4.1 will run on travis, it requires full virtualization and way more resources. This is the only way we can get it working here.

Other option is 3.11, but I prefer 3.9 due to a quick start and low resources utilization.